tests/sympc/protocol/falcon/falcon_test.py

```diff
@@ -26,3 +30,89 @@ def test_exception_malicious_less_parties(get_clients, parties=2) -> None:
 def test_invalid_security_type():
     with pytest.raises(ValueError):
         Falcon(security_type="covert")
+
+
+def test_mul_private_integer(get_clients):
```

Comment:
MAYBE: Also parametrize this

Reply:
Currently, we support only semi-honest protocol for 3 parties so test for the same.